homeassistant/components/hassmpris/translations/es.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Please drop anything but `en.json` as these will be overwritten by lokalize anyways.